### PR TITLE
Update default Theia IDE version to 'next'

### PR DESCRIPTION
**Pull request by sleshchenko**
### What does this PR do?
Update default Theia IDE version to 'next'.
It's needed since commands in Theia IDE 1.0.0 are not working anymore. Error message `No information found` appear.

### What issues does this PR fix or reference?
N/A

#### Release Notes
N/A

#### Docs PR
N/A

### CODE REVIEW

**Comment by sleshchenko** [comment]
I'm not sure that `next` should be default version. @evidolob @benoitf feel free to recommend another one.

@AndrienkoAleksandr Could you please also check that we use right default version for machine-exec plugin.

**Review by mshaposhnik — APPROVED** [review]

File: assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/che/che.properties [hunk 1]
```diff
@@ -619,7 +619,7 @@ che.wsagent.cors.enabled=true
 ## Factory defaults.
 # Editor and plugin which will be used for factories which are created from remote git repository
 # which doesn't contain any Che-specific workspace descriptors (like .devfile of .factory.json)
-che.factory.default_editor=org.eclipse.che.editor.theia:1.0.0
```
Comment (garagatyi) [inline assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/che/che.properties]: Maybe it make sense to also update stacks?

Comment (sleshchenko) [reply assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/che/che.properties]: Good catch. Makes sense. Currently, some of the stacks use `1.0.0` and some `next`. Will update

**Comment by AndrienkoAleksandr** [comment]
> @AndrienkoAleksandr Could you please also check that we use right default version for machine-exec plugin.

 che-machine-exec CHE plugin was bound to the latest image version.

**Comment by monaka** [comment]
@sleshchenko Sorry! When I resolved conflicts on this PR, GitHub web UI pushed it to the branch in your repository (although I don't have write permission for your repo). I believe the patch content itself is reasonable. But please drop my commit if you don't feel good.

**Review by benoitf — APPROVED** [review]

File: ide/che-core-ide-stacks/src/main/resources/stacks.json [hunk 2]
```diff
@@ -214,7 +214,7 @@
         }
       ],
       "attributes": {
-        "plugins": "eclipse/che-machine-exec-plugin/0.0.1",
+        "plugins": "eclipse/che-machine-exec-plugin/next",
         "editor": "eclipse/che-theia/1.0.0"
```
Comment (metlos) [inline ide/che-core-ide-stacks/src/main/resources/stacks.json]: Shouldn't this also be `eclipse/che-theia/next`?

Comment (sleshchenko) [reply ide/che-core-ide-stacks/src/main/resources/stacks.json]: Machine exec should be `0.0.1` and `che-theia` - `next`, and now they are such.

**Pull request merged**